### PR TITLE
Adds compat data for AnimationEffectTimingReadOnly

### PR DESCRIPTION
**Pull request by bennypowers**

### CODE REVIEW

File: api/AnimationEffectTimingReadOnly.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "AnimationEffectTimingReadOnly": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEffectTiming",
```
Comment (teoli2003) [inline api/AnimationEffectTimingReadOnly.json]: This should link to https://developer.mozilla.org/en-US/docs/Web/API/AnimationEffectTimingReadOnly

**Comment by Elchi3** [comment]
This interface has a few properties:
delay
direction
duration
easing
endDelay
fill
iterations
iterationStart

Would you mind adding these as sub features here, too?

**Comment by bennypowers** [comment]
@Elchi3 should that be in a separate PR?

**Comment by Elchi3** [comment]
@bennypowers If you like, you could also do that right here.

**Pull request merged**